l2geth/core/genesis.go

```diff
@@ -496,38 +416,30 @@ func DeveloperGenesisBlock(period uint64, faucet, l1XDomainMessengerAddress comm
 		if err != nil {
 			panic(fmt.Sprintf("Cannot fetch state dump: %s", err))
 		}
-		_, ok := stateDump.Accounts["Lib_AddressManager"]
-		if !ok {
-			panic("Lib_AddressManager not in state dump")
-		}
 	}
-	config.StateDump = &stateDump
+
```

Comment:
I think that the next thing would be to add the `params.ChainConfig` to the state dump json created by `take-dump.ts`

https://github.com/ethereum-optimism/optimism/blob/8d67991aba584c1703692ea46273ea8a1ef45f56/l2geth/params/config.go#L279

Reply:
See how the `geth init` command works: https://github.com/ethereum-optimism/optimism/blob/8d67991aba584c1703692ea46273ea8a1ef45f56/l2geth/cmd/geth/chaincmd.go#L195